tests/tck/features/subgraph/subgraph.IntVid.feature

```diff
@@ -10,86 +10,86 @@ Feature: Integer Vid subgraph
   Scenario: Integer Vid invalid input
     When executing query:
       """
-      GET SUBGRAPH FROM $-.id
+      GET SUBGRAPH WITH PROP FROM $-.id
```

Comment:
It's incompatible. Why don't try `without prop`?

Reply:
To be consistent with the path sentence